.gitignore

```diff
@@ -390,8 +390,8 @@ Temporary Items
 
 =======
 # Local
-docker-compose.yml
-*.env
```

Comment:
`.env` files should not be pushed to git. 

we prepared an `.env.example` to clone from locally :thinking:

Reply:
that env file is loaded by docker-compose in build phase, it evals the docker compose and change the ${VARS} by its equivalent.